network/swarm_test.go

```diff
@@ -45,6 +45,12 @@ var _ = Describe("Swarm service", func() {
 					})
 
 					It("should be able to find most of the nodes in the network after bootstrapping ", func() {
+						// Bootstrap twice
```

Comment:
Bootstrapping once would not always provide sufficient connection between the nodes for the test.

Reply:
After changing this, is dropping the threshold to 1/3 still required?